samples/helloshell/prj_native_posix.conf

```diff
@@ -0,0 +1,40 @@
+# Multi-threaded, with timer support in the kernel
```

Comment:
Move to `samples/helloshell/boards/native_posix.conf` and (re-)set only unsupported features explicitly to "n". That reduce copy and paste and also patch hell of all equal and common configuration entries from the original `samples/helloshell/prj.conf` and also become possible to use another default configuration such as the minimal version from `samples/helloshell/prj-minimal.conf`.

Reply:
done